bower.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "ember-resource",
-  "version": "2.2.0",
+  "version": "2.1.1",
```

Comment:
Im just going to go a big bump then, so that all package managers update to the new version properly.